pkg/provenance/provenance.go

```diff
@@ -147,122 +148,436 @@ func (o ObjectLineage) GetVersions() string {
 	}
 	sort.Ints(s)
 	//get all versions, sort by version, make string array of them
```

Comment:
This comment should be 'make Spec' array of them.

And you can move it above line 146.

Reply:
Actually, "make slice of Specs" as you have on line 172.